README.md

```diff
@@ -111,6 +137,9 @@ end
 ```
 
 - What is mystery6("goodnight moon")?
+- " moon goodnight"
 - What is mystery6("Ada Developers Academy")?
+- " Academy Developers Ada"
 - What is mystery6("Hi, there!")?
+- " there! Hi,"
```

Comment:
minor, but notable: there should be leading spaces at the front of each of these answers: " moon goodnight", " Academy Developers Ada", " there! Hi,"